### PR TITLE
S i m p l i f i c a t i o n

### PR DESCRIPTION
**Pull request by omgmog**
Removes all of the crazy docker stuff that wasn't necessary, moves pages out of the root directory, and some improvements to page title consistency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,8 @@
 
 ## local development
 
-```
-$ bundle install
-$ bundle exec jekyll serve --watch
-```
+Install Jekyll (or the `github-pages` gem) and then simply run:
```
Comment (lpmi-13) [inline README.md:5]: let's add at least a link to the docs on this, cause it'll save me a google next time
https://jekyllrb.com/docs/installation/

File: _layouts/default.html [hunk 2]
```diff
@@ -59,10 +52,10 @@
...
     <a href="/"> / </a>
     {% unless page.hide_page_links %}
-    <a href="/code-of-conduct"> Conduct </a>
-    <a href="/about"> About </a>
-    <a href="/previous-hacks"> Previous Hacks </a>
-    <a href="/podcast"> Podcast </a>
+    <a href="/code-of-conduct/"> Conduct </a>
+    <a href="/about/"> About </a>
+    <a href="/previous-hacks/"> Previous Hacks </a>
+    <a href="/podcast/"> Podcast </a>
```
Comment (lpmi-13) [inline _layouts/default.html:58]: this guy with his style nits :laughing:

File: README.md (hunk 1 above)
Comment (omgmog) [reply README.md]: 👉 c6c52d9

**Review by lpmi-13 — APPROVED** [review]
I will obviously be keeping a docker based branch for myself locally

**Pull request merged**